### ISSUE TITLE
[Project-Organizer] feature request - smart folders for watched projects

### ISSUE THREAD

**Issue by AndrewSallans** [CenterForOpenScience/osf.io#710]
Not necessarily for current release, but would be nice to have a smart folder (like All My Regs and All My Projs) that generates based on projects you Watch.  This would be helpful for reducing steps in creating links for adding to folders.

**Comment by MPNussbaum** [comment]
@GaryKriebel would you please assign me to this? Thank you.

**Comment by lbanner** [comment]
You got it @MPNussbaum -- this one is interesting. Thanks.

**Comment by MPNussbaum** [comment]
@lbanner Thank you.

**Comment by GaryKriebel** [comment]
The PR for this issue is: https://github.com/CenterForOpenScience/osf.io/pull/3320

**Comment by sloria** [comment]
@erinmayhood  Would you mind reviewing this? Is this still a feature we want?

**Comment by nanchen47** [comment]
The pr is done for this problem. Waiting on merge.
